client/src/app/components/Pagination.tsx

```diff
@@ -40,6 +43,38 @@ class Pagination extends Component<PaginationProps, PaginationState> {
         }
     }
 
+    /**
+     * The component mounted.
+     */
+    public componentDidMount(): void {
+        super.componentDidMount();
+
+        window.addEventListener("resize", this.resize.bind(this));
+        this.resize();
+    }
+
+    public resize() {
+        const isMobileViewPort = window.innerWidth < 768
+
+        if ((this.state.isMobile && !isMobileViewPort) ||
+            (!this.state.isMobile && isMobileViewPort)) {
```

Comment:
This condition looks a bit confusing. Consider using `this.state.mobile != isMobileViewPort`, it should have the same effect, if i'm not missing something. We can even extract it to a constant above, something like `shouldUpdatePaginationRange` and merge the two nested ifs.